src/lib/code-sample-editor.ts

```diff
@@ -65,6 +66,10 @@ const serviceWorkerScriptUrl = new URL(
   '../../service-worker.js',
   import.meta.url
 );
+const typescriptWorkerScriptUrl = new URL(
+  '../../typescript-worker.js',
+  import.meta.url
```

Comment:
only problem I have with workers is that they come with lots of caveats in bundling. Perhaps one day it would be nice to have a more-modular approach e.g. give link to worker or something? or is that too much of a security risk?

Reply:
I don't think there's a security problem with that, and we might need to let the worker URL be specified in some other way at some point, especially if Webpack's `import.meta.url` support is still poor. Aiming to get this working with our sites to test this out.